src/engine.c

```diff
@@ -270,6 +276,29 @@ ucschar_strlen (const ucschar* str)
     return p - str;
 }
 
+static gboolean
+check_client_commit (const gchar *version)
+{
+    gboolean client_commit = FALSE;
+    gint len = 0;
+    gint major = 0, minor = 0, micro = 0;
+
+    gchar **versions = g_strsplit_set (version, ".", 3);
+    len = g_strv_length (versions);
+    major = atoi (versions[0]);
+    minor = atoi (versions[1]);
+    micro = atoi (versions[2]);
+    g_strfreev (versions);
+    if (len != 3)
+        return client_commit;
+
+    client_commit = IBUS_CHECK_VERSION (major, minor, micro);
```

Comment:
It doesn't actually check version 1.5.20. It compares compiled ibus version and run time ibus version.

Reply:
@epico Sorry for that. My comment is wrong. So, I removed the comment, but too late.
Could you please implement ibus_hangul_check_ibus_version() which check the ibus version?
Then  we can use this function:

client_commit = ibus_hangul_check_ibus_version(1, 5, 20);

Reply:
No problem.

Okay, I just updated the patch.